src/core/web_api/p2_api.js

```diff
@@ -43,14 +45,16 @@ class Auth0APIClient {
       _sendTelemetry: opts._sendTelemetry === false ? false : true,
       _telemetryInfo: opts._telemetryInfo || default_telemetry,
       state,
-      nonce
+      nonce,
+      scope
     });
 
     this.authOpt = {
```

Comment:
How does this feed into `checkSession` function?

Reply:
everything that goes in the constructor, goes to all the calls

Reply:
checkSession simply calls `logIn` then?

Reply:
no, sorry. they're separate calls. I just fixed logIn as well, since it would have the same issue

adding to the constructor: fixes the reported issue

adding to authOpt: fixes the same issue, but with a login call